packages/2d/src/components/Polygon.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  // PossibleSpacing,
+  BBox,
+  // SpacingSignal,
+} from '@motion-canvas/core/lib/types';
+import {Shape, ShapeProps} from './Shape';
+import {initial, signal} from '../decorators';
+import {SignalValue, SimpleSignal} from '@motion-canvas/core/lib/signals';
+
+function drawPolygon(
```

Comment:
This function could be moved to `src/utils/CanvasUtils.ts`.

Reply:
Done.